src/Validation/Validation.php

```diff
@@ -521,9 +521,11 @@ public static function localizedTime($check, $type = 'datetime', $format = null)
 
     /**
      * Boolean validation, determines if value passed is a boolean integer or true/false.
+     * To check if a value is set specifically to false or True see `inList()`.
      *
      * @param string $check a valid boolean
      * @return bool Success
+     * @see \Cake\Validation\Validation::inList() to mimic isTrue or isFalse checks.
```

Comment:
They are not. inList will work, I am not sure if inList allows type strictness (like PHP's in_array()) atm, but it can probably be made to (e.g. taking 1, 0 only or true/false only, or on/off only etc, or treating 1/true as true, etc etc).

I opted to add docs instead of more functions that do little